conman/routes/tests/test_handlers.py

```diff
@@ -65,6 +66,7 @@ def setUp(self):
         class URLConfRoute(Route):
```

Comment:
Creating the same model repeatedly causes new warnings. I haven't tried to squelch them.